sdk/servicebus/tests.yml

```diff
@@ -5,15 +5,15 @@ stages:
     parameters:
       ServiceDirectory: servicebus
       TestTimeoutInMinutes: 300
-      MatrixReplace:
-        - TestSamples=.*/true
+      TestProxy: true
```

Comment:
The test proxy isn't used in live test pipelines, so we can actually remove this parameter